### PR TITLE
Update .eslintrc.js so that npm run lint behaves as expected

### PR DESCRIPTION
**Pull request by timleslie**
* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -28,8 +28,9 @@ module.exports = {
       'warn',
       'always',
       {
-        ignorePattern: 'pragma|ignored',
+        ignorePattern: '.*:|pragma|ignored|prettier',
         ignoreInlineComments: true,
+        ignoreConsecutiveComments: true,
```
Comment (timleslie) [inline .eslintrc.js]: We generally  don't want to capitalise the first word in a multi-line comment.

File: .eslintrc.js [hunk 2]
```diff
@@ -28,8 +28,9 @@ module.exports = {
       'warn',
       'always',
       {
-        ignorePattern: 'pragma|ignored',
+        ignorePattern: '.*:|pragma|ignored|prettier',
```
Comment (timleslie) [inline .eslintrc.js]: We use commented out `key: value` pairs to indicate config options. We don't want those keys capitalised, hence `.*:`. We also don't want to capitalise `// prettier-ignore` comments.

**Comment by dannyrb** [comment]
@timleslie thanks for taking the time to fine tune these eslint options 🚀

**Pull request merged**